Defra.Cdp.Backend.Api/Program.cs

```diff
@@ -36,10 +36,13 @@
 builder.Configuration.AddEnvironmentVariables();
 
 // Serilog
+builder.Services.AddHttpContextAccessor();
 builder.Logging.ClearProviders();
+var tracingHeader = builder.Configuration.GetValue<string>("Tracing:Header");
 var logger = new LoggerConfiguration()
     .ReadFrom.Configuration(builder.Configuration)
     .Enrich.With<LogLevelMapper>()
+    .Enrich.WithRequestHeader(tracingHeader, tracingHeader)
```

Comment:
`WithCorrelationId` might be a better option here? you give it a header and it puts it in a predictable field, less for tenants to know etc, more guessable.